sql_queries.py

```diff
@@ -71,22 +71,23 @@ def get_table_size(location_id):
 
     # Each cell will be broken up and the value of each section will be
     # compared
+
     for row in cursor.fetchall():
         # Creates a string list seperated by the "-"
         loc = row[0].split('-')
+        if(str(loc[1]).isdigit()):
```

Comment:
again, remember this isn't javascript, you don't need brackets around conditionals